examples/CertificateComplexExample/es6.js

```diff
@@ -13,6 +13,8 @@ import BasicConstraints from "../../src/BasicConstraints.js";
 import ExtKeyUsage from "../../src/ExtKeyUsage.js";
 import CertificateTemplate from "../../src/CertificateTemplate.js";
 import CAVersion from "../../src/CAVersion.js";
+import AttributeTypeDictionnary from "../../src/AttributeTypeDictionnary"
```

Comment:
`AttributeTypeDictionnary ` is no a default class

Please update `AttributeTypeDictionnary` to default class and try to build and run tests locally
```
npm run build:tests
npm test
```